semgrep-core/src/parsing/Parse_target.ml

```diff
@@ -305,12 +305,11 @@ let rec just_parse_with_lang lang file =
   | Lang.Python
   | Lang.Python2
   | Lang.Python3 ->
-      let parsing_mode = lang_to_python_parsing_mode lang in
```

Comment:
Why did you remove the parsing_mode argument?

Reply:
oops, good catch, that was an accident